src/swap/components/Swap.tsx

```diff
@@ -35,10 +38,14 @@ export function Swap({
         )}
         data-testid="ockSwap_Container"
       >
-        <div className="mb-4">
-          <h3 className={text.title3} data-testid="ockSwap_Title">
+        <div className="mb-4 flex items-center justify-between">
+          <h3
+            className={cn(text.title3, 'text-inherit')}
+            data-testid="ockSwap_Title"
+          >
             {title}
           </h3>
+          <div className="flex justify-end">{swapSettings}</div>
```

Comment:
What happen when `swapSettings` is not there?
Should
```
<div className="flex justify-end">
```
Be part of the underline component instead?

Reply:
Good idea, that will be cleaner if SwapSettings is null. 

I've moved that class styling our of `Swap.tsx` and into `SwapSettings.tsx`